config/locales/en.yml

```diff
@@ -108,6 +108,17 @@ en:
         label: "Phone number (for text messages)"
       email:
         label: "Email address"
+    know_nhs_number:
+      title: "Do you know your NHS number?"
+      hint: |
+        An NHS number is a 10 digit number, like 485 777 3456.
```

Comment:
is spacing too big if we use `<p>` tags here?

Reply:
Yes.